### PR TITLE
Host idea.whatwg.org here

### PR DESCRIPTION
**Pull request by annevk**
Fixes https://github.com/whatwg/misc-server/issues/6.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,10 @@ env:
     - ENCRYPTION_LABEL="aec5871fe877"
     - DEPLOY_USER="annevankesteren"
     - SERVER="75.119.197.251"
+    - IDEA_SERVER="165.227.248.76"
     - SERVER_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDP7zWfhJdjre9BHhfOtN52v6kIaDM/1kEJV4HqinvLP2hzworwNBmTtAlIMS2JJzSiE+9WcvSbSqmw7FKmNVGtvCd/CNJJkdAOEzYFBntYLf4cwNozCRmRI0O0awTaekIm03pzLO+iJm0+xmdCjIJNDW1v8B7SwXR9t4ElYNfhYD4HAT+aP+qs6CquBbOPfVdPgQMar6iDocAOQuBFBaUHJxPGMAG0qkVRJSwS4gi8VIXNbFrLCCXnwDC4REN05J7q7w90/8/Xjt0q+im2sBUxoXcHAl38ZkHeFJry/He2CiCc8YPoOAWmM8Vd0Ukc4SYZ99UfW/bxDroLHobLQ9Eh"
```
Comment (foolip) [inline .travis.yml]: The server public key is also different, "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBDt6Igtp73aTOYXuFb8qLtgs80wWF6cNi3/AItpWAMpX3PymUw7stU7Pi+IoBJz21nfgmxaKp3gfSe2DPNt06l8="

Comment (annevk) [inline .travis.yml]: That's a much shorter key. Interesting!

Comment (foolip) [inline .travis.yml]: Apparently eliptic curve crypto has smaller key sizes for the "same" security.

File: deploy.sh [hunk 2]
```diff
@@ -13,5 +13,7 @@ else
     eval `ssh-agent -s`
     ssh-add deploy_key
```
Comment (foolip) [inline deploy.sh]: Hmm wait, this deploy key isn't accepted by idea.whatwg.org yet.

Comment (foolip) [inline deploy.sh]: Added.

**Pull request merged**